### DISCUSSION TITLE
Livewire Integration Status

### DISCUSSION THREAD

**Discussion by danielschweiger** [laravel/nova-issues#3643] [Q&A] [answered]
Hi there,

in the past there was this issue:
https://github.com/laravel/nova-issues/issues/2708

Are there any new updates or plans for integrating Livewire to Nova? Does anybody has done this "his own way" and want's to share his experiences?

Thx :)

**Comment by crynobone** [comment, marked as answer]
> Are there any new updates or plans for integrating Livewire to Nova?

No plans for Livewire supports but Nova 4 will utilize Inertia.js

**Reply by danielschweiger** [reply]
Sounds good. Nova 4 :) How far away is it @crynobone ?
